scargo/file_generators/templates/docker/Dockerfile.j2

```diff
@@ -9,20 +9,17 @@ FROM ${DOCKER_IMAGE_ROOT} as base
 ENV DEBIAN_FRONTEND noninteractive
 
 RUN apt update --fix-missing && \
-    apt -y install python3.8 python3.8-venv python3-pip \
-    binutils git wget \
-    scons build-essential pkg-config \
-    unzip graphviz nano vim && \
+    apt -y install --no-install-recommends python3 python3-pip \
+    git build-essential pkg-config graphviz && \
     apt -y install sudo && \
-    update-alternatives --install /usr/bin/python python /usr/bin/python3 1 && \
-    update-alternatives --install /usr/bin/pip pip /usr/bin/pip3 1
+    rm -rf /var/lib/apt/lists/* && \
```

Comment:
shouldn't be enough to make rm -rf /var/lib/apt/lists/*   at the end of docker file?

Reply:
No, because the layers will still be larger.